waiter/src/waiter/auth/oidc.clj

```diff
@@ -54,12 +54,13 @@
 (defn create-state-code
   "Creates an encoded string of the input state map."
   [state-map password]
-  (utils/map->base-64-string state-map password))
+  (utils/encode-url-safe-b64
```

Comment:
Can we avoid double base64 encoding?

Reply:
Added optional parameter to trigger url-safe base64 encoding/decoding.